src/io/aviso/taxi_toolkit/ui.clj

```diff
@@ -64,6 +64,11 @@
     ("input") (retry #(value el))
     (retry #(text el))))
 
+(defn js-click
+  "Invokes click event on an element using JS API."
+  [el-spec]
```

Comment:
It doesn't support nested elements. You could use variadic fn, like here: https://github.com/AvisoNovate/taxi-toolkit/blob/master/src/io/aviso/taxi_toolkit/ui.clj#L53